src/arviz_plots/plot_collection.py

```diff
@@ -689,8 +689,8 @@ def wrap(
                 {
                     "chart": np.array(fig, dtype=object),
                     "plot": (dims, flat_ax_ary.reshape(plots_raw_shape)),
-                    "row": (dims, flat_row_id.reshape(plots_raw_shape)),
-                    "col": (dims, flat_col_id.reshape(plots_raw_shape)),
+                    "row_index": (dims, flat_row_id.reshape(plots_raw_shape)),
+                    "col_index": (dims, flat_col_id.reshape(plots_raw_shape)),
```

Comment:
I had to rename these variables because this function adds a `column` or `row` coordinate, and coordinates and variables can't have the same name. I like these names better I think, maybe even `column_index`, but this means we need to update this page of the docs too: https://arviz-plots.readthedocs.io/en/latest/tutorials/intro_to_plotcollection.html#customizing-your-plotcollection. Commenting so I don't forget